controller/bindings/label-printer/label_printer.ml

```diff
@@ -16,32 +16,18 @@ let json_of_label label =
   ]
```

Comment:
We could keep the old code using `cohttp` instead of `curl` for `label-printer` and `zetorier`, those changes are confined into a single commit.

Reply:
No, good move to use the curl wrapper everywhere, I think.